lib/utils.js

```diff
@@ -0,0 +1,35 @@
+function isI18nMethodCall(node, methodName) {
+    return node &&
+        node.type === 'CallExpression' &&
+        node.callee.type === 'MemberExpression' &&
+        node.callee.object.type === 'Identifier' &&
+        node.callee.object.name.toLowerCase() === 'i18n' &&
```

Comment:
You are correct. This plugin depends upon us maintaining a convention of how `i18n` gets imported. It might be possible to enforce that convention with eslint as well, but I'm not sure if such a thing already exists. https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/named.md is close.

As for false positives, that too is possible, but if someone did that, it would probably be confusing enough to constitute being a bug.